### PR TITLE
Add in 3DVar FGAT (with time interpolation) for aerosol DA 

### LINKED ISSUE #568: Add FGAT for aerosol DA
Need to explore the use of FGAT for aerosol DA for GDASv17 and see if it is viable. Given that we will still be using Gaussian history for GSI, it may not be feasible and may have to wait until GDASv18. We should explore the benefit of FGAT vs pure 3DVar in the AOD departures, and also see about interpolation and or multiple RESTART writes as a potential option.

### PR DESCRIPTION
**Pull request by CoryMartin-NOAA**
Closes #568 

This PR adds a 3DVar FGAT YAML for aerosol variational DA. Once this is merged, a global-workflow corresponding PR will be made.

### CODE REVIEW

**Comment by CoryMartin-NOAA** [comment]
A quick test does show differences in H(x) between this and the existing YAML template.

**Review by andytangborn — DISMISSED** [review]
It looks fine. The important line (from what I understand) is  tstep: PT3H. I'm assuming everything else is pretty much the same, though I see a few differences from 3dvar_gfs_aero.yaml

File: parm/aero/obs/config/viirs_n20_aod.yaml [hunk 1]
```diff
@@ -14,6 +14,7 @@ obs space:
   channels: 4
 get values:
   interpolation method: barycentric
```
Comment (guillaumevernieres) [inline parm/aero/obs/config/viirs_n20_aod.yaml:16]: Ha! I wasn't aware this was working @CoryMartin-NOAA. Nice!

Comment (CoryMartin-NOAA) [reply parm/aero/obs/config/viirs_n20_aod.yaml:16]: Well it wasn't, but there is an OOPS bugfix PR so I figured might as well.

**Review by guillaumevernieres — APPROVED** [review]
:tada:

**Pull request merged**